tests/python/mints14/input.py

```diff
@@ -0,0 +1,73 @@
+#! Check for correctness of ESP values. The ESP values are calculated using one or four threads
+#! The one thread values are checked against the four thread values. The one thread values are 
+#! also checked against the reference values (1 thread values computed, when generating this test).
+#! Caution: The reference values are not obtained using an actual physical reference, but rather
+#! generated by Psi4 at one point in time.
+
+import psi4
+import psi4.core as p4c
+import numpy as np
+
+psi4.set_output_file("output.dat", False)
+
+#H2O with minimal basis
+h2o = psi4.geometry("""
+O
+H 1 0.96
+H 1 0.96 2 104.5
+""")
+
+
+ene, wfn = psi4.energy('scf/STO-3G', return_wfn = True)
+myepc = p4c.ESPPropCalc(wfn)
+
+# Number of points to evaluate the ESP on per dimension (reference values below were generated with 5)
+numpoints_per_dim = 5
+points = np.array([ [float(x),float(y),float(z) ] for x in range(0,numpoints_per_dim) for y in range(0,numpoints_per_dim) for z in range(0,numpoints_per_dim) ])
+psi4_matrix = p4c.Matrix.from_array(points)
+
+# We calculate the ESPs with 1 thread
+psi4.set_num_threads(1, quiet = True)
+esps_1threads = np.array(myepc.compute_esp_over_grid_in_memory(psi4_matrix))
+# We calculate the ESPs with 4 threads
+psi4.set_num_threads(4, quiet = True)
```

Comment:
A complication is that the whole test suite is often run `-n<num_cores>`, so extra threads aren't available. But since this is focused on launching the threads, not checking for speedup, I think it'll be ok.